src/main/java/org/sunbird/user/controller/UserUtilityController.java

```diff
@@ -22,4 +21,10 @@ public ResponseEntity<Map<String, Object>> updateLogin(@RequestBody LastLoginInf
 			throws NumberFormatException {
 		return new ResponseEntity<>(userUtilService.updateLogin(userLoginInfo), HttpStatus.OK);
 	}
+
+	@PostMapping("/user/v1/autocomplete")
+	public ResponseEntity<Map<String, Object>> autoSearchUser(@RequestParam String key)
+			throws NumberFormatException, IOException {
+		return new ResponseEntity<>(userUtilService.autoSearchUser(key), HttpStatus.OK);
+	}
```

Comment:
Post method should have request body. pass the request body to userUtilService.
Need one more GET method which reads "key" from query param